webui/react/src/utils/wait.ts

```diff
@@ -11,7 +11,7 @@ export interface WaitStatus {
 }
 
 export const commandToEventUrl = (command: Command | CommandTask): string => {
-  const kind = isCommandTask(command) ? command.type : command.kind;
+  const kind = isCommandTask(command) ? command.type : command.type;
```

Comment:
nit: if you're renaming kind to type elsewhere, might as well rename this variable too